src/en/space-station-14/mapping/guides/general-guide.md

```diff
@@ -17,9 +17,9 @@ If you've read "Workflow" and are looking for a refresher:
    
 2. If desired, `mappingclientsidesetup` to use the default mapping setup.
 
-3. Make all your changes.
+3. Make all mapping changes.
    
-4. If you have created new rooms, run `fixgridatmos GID` before saving.
+4. If new rooms were created, run `fixgridatmos GID`.
```

Comment:
The passive voice was used. The passive voice is a bit weaker and less clear than the active voice. It was also good to clarify that this should be run before saving.